extensions/pyRevitTools.extension/pyRevit.tab/Analysis.panel/Tools.stack/Inspect.pulldown/Model Checker.pushbutton/script.py

```diff
@@ -31,19 +31,33 @@
             "#4d4d4d","#fff0d9","#ffc299","#ff751a","#cc5200","#ff6666","#ffd480","#b33c00","#ff884d","#d9d9d9","#9988bb","#4d4d4d","#e97800","#a6c844",
             "#4d4d4d","#fff0d9","#ffc299","#ff751a","#cc5200","#ff6666","#ffd480","#b33c00","#ff884d","#d9d9d9","#9988bb","#4d4d4d","#e97800","#a6c844",]
 
-
-# list of Warnings rated as critical (warnings which could distort quantities and schedules)
-criticalWarnings = ['Elements have duplicate "Type Mark" values',
-    'There are identical instances in the same place',
-    'Room Tag is outside of its Room',
-    'Multiple Rooms are in the same enclosed region',
-    'Multiple Areas are in the same enclosed region',
-    'One element is completely inside another',
-    'Room is not in a properly enclosed region',
-    'Room separation line is slightly off axis and may cause inaccuracies',
-    'Area is not in a properly enclosed region',
-    "Rectangular opening doesn't cut its host",
-    'Elements have duplicate "Number" values',]
+# citical warnings Guids
```

Comment:
I changed Critical warning list to list of GUIDs. Now it is compatible to other laguages than english. I am not sure if there is some better identifier than GUIDs.